gosushi_server/classes/game.js

```diff
@@ -28,6 +28,25 @@ class Game {
     this.uramakiStanding = { value: 1 };
     this.specialActions = [];
     this.gameStarted = false;
+    this.isGameOver = false;
+  }
+
+  newGame(menu, playerNum, hostPlayer) {
+    this.deck = new Deck(menu, playerNum);
+    this.numPlayers = playerNum;
+    this.round = 1;
+    this.hands = [];
+    this.hostPlayer.name = hostPlayer;
+    this.playedTurn = 0;
+    this.uramakiCountMap = {};
+    this.uramakiStanding.value = 1;
+    this.specialActions = [];
+    this.gameStarted = false;
+    this.isGameOver = false;
+
+    this.players.forEach(p => {
```

Comment:
Do you reset this.players? We should be ensuring this.players has length this.numPlayers.